src/main/java/org/reaktivity/reaktor/Reaktor.java

```diff
@@ -110,6 +111,11 @@ public void detach(
         }
     }
 
+    public void visit(Consumer<Controller> action)
+    {
+        this.controllersByKind.values().forEach(action);
+    }
+
```

Comment:
Can you explain how you are using this functionality?

It may be more appropriate to expose the nukleus names and controller kinds instead.